ngraph/test/runtime/interpreter/reference/scatter_nd_update.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/coordinate_transform.hpp"
+#include "ngraph/shape.hpp"
+
+using namespace ngraph;
+
+namespace referenceTest
```

Comment:
Why namespace is "referenceTest"?

Reply:
It's just namespace for reference implementations that using only for tests and don't used in ngraph backend

Reply:
ok

Reply:
Can I ask you to add all reference implementations in the namespace as in original nGraph? It should help in future copy past these implementations in the nGraph library.

```
namespace ngraph
{
    namespace runtime
    {
        namespace reference
        {
```

Reply:
changed namespace